packages/pds/src/db/tables/post-embed-external.ts

```diff
@@ -5,7 +5,7 @@ export interface PostEmbedExternal {
   uri: string
   title: string
   description: string
-  thumbCid: string
+  thumbCid?: string
```

Comment:
real quick: this should be `string | null`. kysely returns as `null` not `undefined`